### PR TITLE
PM-1281 Changed HttpStatus From 302 to 200 resume/method

### PR DESCRIPTION
**Pull request by DavideAmatoNexi**
#### List of Changes

Changed HttpStatus From 302 to 200 resume/method

#### Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have created or updated an API interface.
- [x] I have updated the OpenAPI spec
- [ ] I have created or updated a database table
  - [ ] I have updated both the Oracle and Postgres migrations
- [x] My PR is cohesive and self-contained (rule of thumb: less than 250 changed lines) 

### Link to story

https://pagopa.atlassian.net/browse/PM-1281

### CODE REVIEW

File: api-specs/api_spec_PGS.yaml [hunk 1]
```diff
@@ -689,8 +689,8 @@ paths:
               $ref: '#/components/schemas/CreditCardResumeRequest'
         required: true
       responses:
-        "302":
-          description: "FOUND, Redirect to url"
+        "200":
+          description: "OK"
```
Comment (giovanniberti) [inline api-specs/api_spec_PGS.yaml:693]: Can you put a more descriptive explanation of what a 200 response entails?

**Comment by giovanniberti** [comment]
Please also fix formatting in the PR description 🙂

File: api-specs/api_spec_PGS.yaml (hunk 1 above)
Comment (DavideAmatoNexi) [reply api-specs/api_spec_PGS.yaml]: Done

**Review by giovanniberti — APPROVED** [review]
Please fix formatting in the PR description

**Pull request merged**